massa-graph/src/block_graph.rs

```diff
@@ -1188,7 +1204,28 @@ impl BlockGraph {
     pub fn get_export_block_status(&self, block_id: &BlockId) -> Option<ExportBlockStatus> {
         self.block_statuses
             .get(block_id)
-            .map(|block_status| block_status.into())
+            .map(|block_status| match block_status {
```

Comment:
Is there a reason to not use the into anymore ?

Reply:
Yes, the need to retrieve the block from storage(which is not available from inside the implementation of `into`).